### PR TITLE
Add keep-frontmost extension

### PR DESCRIPTION
**Pull request by Mmehdi06**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/keep-frontmost/package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "keep-frontmost",
+  "title": "Keep Frontmost",
```
Comment (pernielsentikaer) [inline extensions/keep-frontmost/package.json:4]: Can we think of a better name, it's kinda not saying what it does

File: extensions/keep-frontmost/package.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  "author": "Mmehdi06",
+  "categories": [
+    "System"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Keep Frontmost",
+      "subtitle": "Close All Apps Except Frontmost",
```
Comment (pernielsentikaer) [inline extensions/keep-frontmost/package.json:16]: Subtitle should not be used to describe, can we add a better title?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Just added a few more comments for you to look at 🙂

File: extensions/keep-frontmost/package.json (hunk 1 above)
Comment (Mmehdi06) [reply extensions/keep-frontmost/package.json:4]: What do you think about SingleFocus ?

File: extensions/keep-frontmost/package.json (hunk 2 above)
Comment (Mmehdi06) [reply extensions/keep-frontmost/package.json:16]: what about this?

title: Focus on active app
subtitle: SingleFocus

File: extensions/keep-frontmost/package.json (hunk 1 above)
Comment (pernielsentikaer) [reply extensions/keep-frontmost/package.json:4]: That could work as title for sure

File: extensions/keep-frontmost/package.json (hunk 2 above)
Comment (Mmehdi06) [reply extensions/keep-frontmost/package.json:16]: yes you are right, i also thought about "Clear Distractions"

Comment (Mmehdi06) [reply extensions/keep-frontmost/package.json:16]: It realy is I'll think about it further and come back to you later today. Thank you for the feedback.

**Pull request closed without merging**